src/script/view_model/PanelViewModel.ts

```diff
@@ -132,8 +133,10 @@ export class PanelViewModel {
     const conversationState = container.resolve(ConversationState);
 
     this.elementId = 'right-column';
+
     this.repositories = repositories;
     this.mainViewModel = mainViewModel;
+    this.isFederated = mainViewModel.isFederated;
```

Comment:
The bug was here: the view was trying to access a `isFederated` property which was `undefined`.